CineCompass/Model/MovieManager.swift

```diff
@@ -49,6 +83,16 @@ final class MovieManager {
             print("error decoding JSON")
             return nil
         }
-        
+    }
+    
+    func parseMovieDetailsData(_ data: Data) -> MovieDetailsData? {
```

Comment:
name convention tavsiyesi.

Eğer serverden gelen response varsa, file isminin sonunan `...Response` ekleyerek struct in okunabilirligini artirabilirsin.
e.i. `MovieDetailsDataRequest` or `MovieDetailsDataResponse`